glusterd2/commands/volumes/volume-create.go

```diff
@@ -251,3 +270,17 @@ func volumeCreateHandler(w http.ResponseWriter, r *http.Request) {
 func createVolumeCreateResp(v *volume.Volinfo) *api.VolumeCreateResp {
 	return (*api.VolumeCreateResp)(createVolumeInfoResp(v))
 }
+
+func getRedundancy(disperse uint) uint {
```

Comment:
I'll wait for someone worked on EC to review this part.

Reply:
@aspandey Can you please review it?